ztp/source-crs/DefaultCatsrc.yaml

```diff
@@ -11,3 +11,6 @@ spec:
     image: $imageUrl
     publisher: Red Hat
     sourceType: grpc
+    updateStrategy:
+        registryPoll:
+            interval: 1h
```

Comment:
Please ensure there is a newline at the end of the file to avoid github flagging this.

Reply:
Done